impeller/renderer/backend/vulkan/shader_function_vk.h

```diff
@@ -25,8 +26,10 @@ class ShaderFunctionVK final
   friend class ShaderLibraryVK;
 
   vk::UniqueShaderModule module_;
+  std::weak_ptr<DeviceHolder> device_;
```

Comment:
Rename to `device_holder_` for consistency

Reply:
done